modules/007-registrypackages/images/d8-shutdown-inhibitor/src/pkg/app/containerd/list_pods.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package containerd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+)
+
+func ListPods(ctx context.Context) (*PodList, error) {
+	crictl := exec.CommandContext(ctx, "crictl", "pods", "-o", "json")
```

Comment:
why we use crictl here? do we can use kube-client here?

I am seeing that we already use klient-go here https://github.com/deckhouse/deckhouse/pull/12241/files#diff-18151c9ff0bfe3879f0aac611131c8b88981d4bbf896a1dbe3a5066cc0448ef2R44

Reply:
Containerd package is a first implementation and not used anymore, I've deleted it.

There is no client-go here, this component execs kubectl command. It is PoC, we can update to use client-go later.

Reply:
This package was removed